airflow/providers/google/ads/hooks/ads.py

```diff
@@ -195,6 +229,22 @@ def _get_config(self) -> None:
 
         self.google_ads_config = conn.extra_dejson["google_ads_client"]
 
+    def _get_authentication_method(self) -> None:
```

Comment:
Should we rename it as something like `_determine_authentication_method`? `_get...` without returning anything seems to be confusing to me

Reply:
@Lee-W Fully agree that your suggestion makes sense. Reason for using `_get..` was to conform with the already existing method _get_config, but they are a bit different in nature, so let me refactor